src/warc2zim/statics/wombat_setup.js

```diff
@@ -0,0 +1,197 @@
+// This file setup a wbinfo dictionnary to configure a wombat instance.
+// It mainly export one function `getWombatInfo` which takes context information
+// and returns a configuration dictionnary to pass to wombat.
+
+const getWombatInfo = (function () {
+  // Reduce "complex" url to our simplyfied ones.
+  // ! MUST be in sync with the python code (at zim creation time)
+  let reduce = function (path) {
```

Comment:
I don't get why this is not a `const`?

Reply:
Keyboard mistake. I swear (please trust me), I've typed `const` but the keyboard writes `let` instead.

This is fixed.